### PR TITLE
[BET-63] Desktop support for WhatsApp share buttons

### PR DESCRIPTION
**Pull request by 00salmon**
## Description of the change

> WhatsApp can now open on the desktop, so we have added support for that option in raisins.

## Type of change

- [x] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation or Development tools (readme, specs, tests, code formatting)

## Links

- Jira issue number: (PUT IT HERE)
- Process.st launch checklist: (PUT IT HERE)

### Development

- [ ] [Prettier](https://www.npmjs.com/package/prettier) was run (if applicable)
- [ ] The behaviour changes in the pull request are covered by specs
- [ ] All tests related to the changed code pass in development

### Paperwork

- [ ] This pull request has a descriptive title and information useful to a reviewer
- [ ] This pull request has a Jira number
- [ ] This pull request has a Process.st launch checklist

### Code review

- [ ] Changes have been reviewed by at least one other engineer
- [ ] Security impacts of this change have been considered

### CODE REVIEW

File: packages/vanilla-components/CHANGELOG.md [hunk 1]
```diff
@@ -7,32 +7,42 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 
 ## [1.2.3] - 2022-03-22
 
 ### Changed
 
 - \<sqh-referral-list> show-referrer prop now prevents referrer from being queried when set to "false"
 
-[unreleased]: https://github.com/saasquatch/program-tools/compare/vanilla-components@1.2.4...HEAD
+[unreleased]: https://github.com/saasquatch/program-tools/compare/vanilla-components@1.2.5...HEAD
+[1.2.4]: https://github.com/saasquatch/program-tools/releases/tag/%40saasquatch%2Fvanilla-components%401.2.5
```
Comment (DerekSiemens) [inline packages/vanilla-components/CHANGELOG.md:45]:
```suggestion
[1.2.5]: https://github.com/saasquatch/program-tools/releases/tag/%40saasquatch%2Fvanilla-components%401.2.5
```

File: packages/vanilla-components/CHANGELOG.md [hunk 2]
```diff
@@ -7,32 +7,42 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.2.5] - 2022-09-27
```
Comment (DerekSiemens) [inline packages/vanilla-components/CHANGELOG.md:10]: Flagging date update needed

Comment (00salmon) [inline packages/vanilla-components/CHANGELOG.md:10]:
```suggestion
## [1.2.5] - 2022-09-28
```

Comment (00salmon) [inline packages/vanilla-components/CHANGELOG.md]:
```suggestion
## [1.2.5] - 2022-09-28
```

**Review by DerekSiemens — APPROVED** [review]

**Pull request merged**